src/services/TeamAssignmentService.ts

```diff
@@ -0,0 +1,37 @@
+import { NewActiveMatchInput } from "../repositories/ActiveMatchRepository/types";
+import { PlayerInQueue } from "../repositories/QueueRepository/types";
+import { Team } from "../types/common";
+
+export async function createRandomTeams(ballchasers: PlayerInQueue[]): Promise<Array<NewActiveMatchInput>> {
+  const sortedBallChaser = ballchasers.sort((o, b) => (o.mmr > b.mmr ? -1 : 1));
```

Comment:
Very small note, honestly you don't even need to change it if you want, but you can just do `o.mmr - b.mmr` and get the same result.

Reply:
I fixed to clean it up.  No need to have unnecessary logic if not needed.